adafruit_sgp40/voc_algorithm.py

```diff
@@ -0,0 +1,809 @@
+# SPDX-FileCopyrightText: Copyright (c) 2010 DFRobot Co.Ltd (http://www.dfrobot.com)
+#
+# SPDX-License-Identifier: MIT
+"""
+`voc_algorithm`
+================================================================================
+
+Class and algorithm to convert Sensirion sgp40 raw reading to indexed voc readings.
+
+
+* Author(s): yangfeng
+
+"""
+
+VOCALGORITHM_SAMPLING_INTERVAL = 1.0
+VOCALGORITHM_INITIAL_BLACKOUT = 45.0
+VOCALGORITHM_VOC_INDEX_GAIN = 230.0
+VOCALGORITHM_SRAW_STD_INITIAL = 50.0
+VOCALGORITHM_SRAW_STD_BONUS = 220.0
+VOCALGORITHM_TAU_MEAN_VARIANCE_HOURS = 12.0
+VOCALGORITHM_TAU_INITIAL_MEAN = 20.0
+VOCALGORITHM_INITI_DURATION_MEAN = 3600.0 * 0.75
+VOCALGORITHM_INITI_TRANSITION_MEAN = 0.01
+VOCALGORITHM_TAU_INITIAL_VARIANCE = 2500.0
```

Comment:
can make these `const()` have the name start with `_` as well - this will reduce the memory usage